docs/positioning.md

```diff
@@ -0,0 +1,162 @@
+# _positioning.scss
+_File includes function and mixins which help with positioning different type of elements_
+
+### List of content:
+
+- [Mixin size](#function-size) (bootstrap 4.0.0-beta)
+- [Mixin clearfix](#mixin-clearfix) (bootstrap 4.0.0-beta)
+- [Function z](#function-z)
+- [Mixin pseudo](#mixin-pseudo)
+- [Mixin position-variant](#mixin-position-variant)
+- [Mixin absolute](#mixin-absolute)
+- [Mixin fixed](#mixin-fixed)
+- [Mixin relative](#mixin-relative)
+- [Mixin sticky](#mixin-sticky)
+
+
+### Default variables
+_Default variables declared in file to be used in mixins._
+
+```scss
+$z-indexes: (
+  'modal',
+  'header',
+  'body',
+  'footer'
+) !default;
+```
+
+## Mixin size
+
+### Description
+_Bootstrap helper which helps setting width and height of element_
+
+### Parameters
+- `$width` - variable describing width (***required***)
+- `$height` - variable describing height (as default it's used ```$width```)
+
+
+## Mixin clearfix
+
+### Description
+_Bootstrap mixin which helps with wrapping containers_
```

Comment:
Bootstrap helper. Used on a container of which all the children are floating. If you would not use this, the container would not get any height (except in IE)